includes/export/abstract-wc-csv-exporter.php

```diff
@@ -330,6 +330,8 @@ public function escape_data( $data ) {
 			$data = "'" . $data;
 		}
 
+		$data = str_replace( '"', '""', $data );
```

Comment:
https://stackoverflow.com/questions/17808511/properly-escape-a-double-quote-in-csv